Tools/scripts/run_flake8.py

```diff
@@ -19,16 +19,18 @@
 class Flake8Checker(object):
     def __init__(self):
         self.retcode = 0
+        self.files_to_check = []
 
     def progress(self, string):
         print("****** %s" % (string,))
 
-    def check(self, filepath):
-        self.progress("Checking (%s)" % filepath)
-        retcode = subprocess.call(["flake8", filepath])
-        if retcode != 0:
-            self.progress("File (%s) failed with retcode (%s)" %
-                          (filepath, retcode))
+    def check(self):
+        for path in self.files_to_check:
+            self.progress("Checking (%s)" % path)
+        ret = subprocess.run(["flake8", "--show-source"] + self.files_to_check,
+                             stdout=subprocess.PIPE, stderr=subprocess.STDOUT, text=True)
```

Comment:
We may hit limits on command-line-length with this.  But it's not an issue now, so I think we're OK.